spec.emu

```diff
@@ -62,23 +62,56 @@ contributors: Google, Ecma International
         The IsValidCodeForDateTimeField abstract operation verifies that the given _code_ is valid.
       </p>
       <emu-alg>
-        1. If _code_ is in the following list, return *true*.
-        <ul>
-          <li>era</li>
-          <li>year</li>
-          <li>quarter</li>
-          <li>month</li>
-          <li>weekOfYear</li>
-          <li>weekday</li>
-          <li>day</li>
-          <li>dayPeriod</li>
-          <li>hour</li>
-          <li>minute</li>
-          <li>second</li>
-          <li>timeZoneName</li>
-        </ul>
+        1. If _code_ is listed in <emu-xref href="#table-validcodefordatetimefield"></emu-xref>, return *true*.
         1. Return *false*.
       </emu-alg>
+
+      <emu-table id="table-validcodefordatetimefield">
+        <emu-caption>Codes For Date Time Field of DisplayNames</emu-caption>
+        <table class="real-table">
+          <thead>
+            <tr>
+              <th>Code</th>
+            </tr>
+          </thead>
+          <tr>
+            <td>`"era"`</td>
```

Comment:
in a follow up, we can expand the tables to add a descriptive cell saying what each code means.